meta-aos-rcar-gen4-dom0/recipes-core/runx/runx_git.bb

```diff
@@ -0,0 +1,60 @@
+HOMEPAGE = "https://github.com/lf-edge/runx"
+SUMMARY = "runx stuff"
+DESCRIPTION = "Xen Runtime for OCI"
+
+SRCREV_runx = "0c7edb3453398d7a0c594ce026c9c1e93c2541cc"
+
+SRC_URI = "\
+	  git://github.com/lf-edge/runx;nobranch=1;name=runx \
+	  file://0001-files-create-Update-create-script-to-start-unikernel.patch \
+	  "
+
+SRC_URI[md5sum] = "ce9b2d974d27408a61c53a30d3f98fb9"
+SRC_URI[sha256sum] = "bf338980b1670bca287f9994b7441c2361907635879169c64ae78364efc5f491"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=945fc9aa694796a6337395cc291ddd8c"
+
+S = "${WORKDIR}/git"
+PV = "v1.1-git${SRCREV_runx}"
+
+inherit pkgconfig
+
+RDEPENDS_${PN} += " jq bash"
+RDEPENDS_${PN} += " xen-tools-xl socat daemonize"
+
+RUNX_USE_INTERNAL_BUSYBOX ?= ""
+
+do_compile() {
+    export CC="${CC}"
+    export LD="${LD}"
+    export CFLAGS="${HOST_CC_ARCH}${TOOLCHAIN_OPTIONS} ${CFLAGS}"
+    export LDFLAGS="${TOOLCHAIN_OPTIONS} ${HOST_LD_ARCH} ${LDFLAGS}"
+    export HOSTCFLAGS="${BUILD_CFLAGS} ${BUILD_LDFLAGS}"
+    export CROSS_COMPILE="${TARGET_PREFIX}"
+    export ARCH="%{TARGET_PREFIX}"
+    cd ${S}/sendfd
+    make
+}
+
+do_install() {
+    install -d ${D}${bindir}
+    install -m 755 ${S}/runX ${D}${bindir}
+
+    install -d ${D}${datadir}/runX
+    install -m 755 ${S}/files/start ${D}/${datadir}/runX
+    install -m 755 ${S}/files/create ${D}/${datadir}/runX
+    install -m 755 ${S}/files/state ${D}/${datadir}/runX
+    install -m 755 ${S}/files/delete ${D}/${datadir}/runX
+    install -m 755 ${S}/files/pause ${D}/${datadir}/runX
+    install -m 755 ${S}/files/mount ${D}/${datadir}/runX
+    install -m 755 ${S}/files/serial_start ${D}/${datadir}/runX
+    install -m 755 ${S}/sendfd/sendfd ${D}/${datadir}/runX
+}
+
+deltask compile_ptest_base
+
+FILES_${PN} += "${bindir}/* ${datadir}/runX/*"
```

Comment:
`=` better suits here.

Reply:
I prefer using += because lets say I want to add new files and missed this 57th line. I would add FILES = .... to the beginng of the file. In this case my variable would be overrided. I agree - = suits better. If my motivation isn't enough - I would update.